lib/reporters/mini.js

```diff
@@ -70,19 +70,16 @@ MiniReporter.prototype.prefix = function (str) {
 };
 
 MiniReporter.prototype._test = function (test) {
-	var title;
+	var title = test.title;
 
 	if (test.todo) {
-		title = colors.todo('- ' + test.title);
 		this.todoCount++;
```

Comment:
As I recall, the API does track this, but it's not provided until the `finish` event. The mini reporter is the only reporter that cares about the live count.